core/store/models/eth_test.go

```diff
@@ -368,3 +270,23 @@ func TestHead_EarliestInChain(t *testing.T) {
 
 	assert.Equal(t, int64(1), head.EarliestInChain().Number)
 }
+
+func TestTxReceipt_ReceiptIndicatesRunLogFulfillment(t *testing.T) {
+	tests := []struct {
+		name string
+		path string
+		want bool
+	}{
+		{"basic", "../../services/eth/testdata/getTransactionReceipt.json", false},
```

Comment:
It would be nice to copy these testdata directories locally, as I believe that's the idiomatic practice. Having said that, I know our usage isn't perfect everywhere and it may just be better to circle back and do that in another PR.